src/main/java/seedu/duke/profile/Profile.java

```diff
@@ -21,12 +27,12 @@
      * @throws InvalidCharacteristicException If a value of <= 0 is provided for height or weight
      * @throws NullCharacteristicException    When the input name is null or ""
      */
-    public Profile(String name, double height, double weight)
+    public Profile(String name, double height, double weight, int calorieGoal)
             throws InvalidCharacteristicException {
         setName(name);
         setHeight(height);
         setWeight(weight);
-        setCalorieGoal(0); //Initialize to 0 first
+        setCalorieGoal(calorieGoal); //Initialize to 0 first
```

Comment:
i think that the comment here can be removed since calories goal will not be initialize to 0 first :)